tests/pytests/unit/modules/test_napalm_bgp.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
Please move all of these tests under their own python package, ie, `tests/pytests/unit/modules/test_napalm_bgp.py` -> `tests/pytests/unit/modules/napalm/test_bgp.py`